app/controllers/application_controller.rb

```diff
@@ -20,7 +20,7 @@ def authenticate_with_identity
       secure: true,
     }
 
-    redirect_to identity_authenticate_url(identity_state)
+    redirect_to identity_authenticate_url(identity_state), allow_other_host: true
```

Comment:
Rails 7 prevents redirects to external hosts by default. https://api.rubyonrails.org/v7.0.8.4/classes/ActionController/Redirecting.html#method-i-redirect_to